bela/render.cpp

```diff
@@ -0,0 +1,263 @@
+/*------------------------------------------------------------------------------
+ *
+ *  render.cpp
+ *
+ *  This exemple sends "Hello World !" when it starts and decodes it at the same
+ *  time.
+ *
+ *  For full information on usage and licensing, see https://chirp.io/
+ *
+ *  Copyright © 2011-2019, Asio Ltd.
+ *  All rights reserved.
+ *
+ *----------------------------------------------------------------------------*/
+
+#include <Bela.h>
+#include <stdlib.h>
+#include <string.h>
+#include <string>
```

Comment:
This doesn't look right

Reply:
The second one is the C++ one to get the `std::string` type